PyDSS/helics_interface.py

```diff
@@ -1,18 +1,19 @@
+from pydantic import BaseModel, ValidationError, validator
+from typing import List, Optional, Any, Union, Dict
+from enum import Enum
 import logging
 import helics
 import os
+import re
 
-from PyDSS.pyContrReader import pyExportReader, pySubscriptionReader
-from PyDSS.pyLogger import getLoggerTag
-from PyDSS.simulation_input_models import SimulationSettingsModel
-
-
-class helics_interface:
 
-    n_states = 5
-    init_state = 1
+from PyDSS.pyContrReader import pyExportReader
+from PyDSS.simulation_input_models import SimulationSettingsModel
+from PyDSS.common import SUBSCRIPTIONS_FILENAME, ExportMode
+from PyDSS.pyLogger import getLoggerTag
+from PyDSS.utils.utils import load_data
 
-    type_info = {
+type_info = {
```

Comment:
Can you make this all-caps so that readers know it is a global “constant"?

Reply:
done